volumes/vds/readme.md

```diff
@@ -0,0 +1,468 @@
+
+  ![Division_id 8001 sensors with non-zero volumes in August 2023:  ](<exploration/div8001-aug2023-non zero volume.PNG>)  
+
+</div>
+
+**August 2023 division_id 8001 Summary:**
+| det_type         | num_sensors | avg_daily_volume | median_daily_volume | max_daily_volume | total_volume |
+| ---------------- | ----------- | ---------------- | ------------------- | ---------------- | ------------ |
+| Detector (DET) | 9541        | 349.6113         | 0                   | 9398             | 1.03E+08     |
+| Preemption (PE) | 79          | 0.234381         | 0                   | 25               | 574          |
+| Special Function (SF) | 435         | 0.003189         | 0                   | 1                | 43           |
+
+The data from "special function" and "preemption" detectors seem extremely dubious. They both have less than 1 average detection per sensor per day. 
+The regular detectors (DET) may have some utility but it is hard to tell with the prevelance of zeros (the median daily detector volume is zero). 
+
+## How do I access it? Where is the OpenData (if it exists)?
+- Cleaned data is stored in `bigdata`, `vds` schema. 
+- Raw data is stored in `transnomis` postgres database `public` schema, which can be accessed by remoting in to terminal server and using connection details stored in Airflow. 
+- [ITS Central](https://itscentral.corp.toronto.ca/) is helpful for exploring the detector inventory. Jesse can help you gain access.
+- OpenData: RESCU data was never added to OpenData due to data quality concerns, however this decision should be reviewed with improved availability due to new pipeline and ability to formulate better data quality metrics. In particular `veh_speeds_15min` could be released in a similar pattern to existing `wys` OpenData release and may be of interest to public. 
+
+## Data Availability
+- Tables `vds.raw_vdsdata` and `vds.raw_vdsvehicledata` and related summary tables (`counts_15min`, `counts_15min_bylane`, `veh_length_15min`, `veh_speeds_15min`) have data from 2021-11-01 and beyond pulled from ITSC using the new process described here.  
+- Data for table `vds.counts_15min` from 2017-01-01 to 2021-10-31 was backfilled from `rescu.volumes_15min`, and contains nulls for certain columns: `lanes`, `num_distinct_lanes`, `num_obs`. Only highway RESCU sensors were included in this old schema. 
+- There are various data outages, although many are eliminated under this new vds pipeline.
+
+## How was it aggregated & filtered? What pitfalls should I avoid?
+- Detectors/time periods with no records will not appear in summary tables.
+- Some detectors have come and gone, for example the Allen. Check the sensor you want is available in the time period. 
+- Zeros/nulls/missing records are all considered zeros when aggregating to `counts_15min` and `counts_15min_bylane` in line with logic under previous `RESCU` pipeline. Aggregation method can be adjusted/examined by going back to `raw_vdsdata` table. 
```

Comment:
Can add a reference here to this past work, until more automated checks and transformation are in place
https://github.com/CityofToronto/bdit_data-sources/blob/master/volumes/rescu/README.md#6--how-often-are-data-quality-assessment-processes-for-the-data-undertaken